arch/xtensa/src/esp32s2/esp32s2_spi.c

```diff
@@ -233,11 +352,18 @@ static struct esp32s2_spi_priv_s esp32s2_spi2_priv =
 {
   .spi_dev     =
     {
-      .ops     = &esp32s2_spi2_ops
+      .ops = &esp32s2_spi2_ops
     },
   .config      = &esp32s2_spi2_config,
   .refs        = 0,
-  .lock        = NXMUTEX_INITIALIZER,
+  .exclsem     = SEM_INITIALIZER(0),
```

Comment:
@acassis 
Do you want to revert the change introduced https://github.com/apache/incubator-nuttx/pull/6965?
I think the changes are not DMA related and perhaps your code base is too old.